sections/sprint-3/dead-reckoning.tex

```diff
@@ -0,0 +1,84 @@
+In this example, dead reckoning would be a linear problem where the position, velocity and acceleration can be used to predict where the player will move to in the future. 
+The dead reckoned position for a specific time $Q_t$ in this example can be calculated by:
+\begin{displaymath}
+    Q_t = P'_0 + V'_0T + \frac{1}{2}A'_0T^2
+\end{displaymath}
+where $ P'_0 $ is the position, $ V'_0 $ is the velocity, $ A'_0 $ is the acceleration and $T$ is the time \autocite{DeadReckoning}.
+\begin{figure}[H]
+    \centering
+    \includegraphics[width=0.6\linewidth]{dead_reckoning/dead_reckoning_example_2.PNG}
+    \caption{A new update about the player's state is received from the server}
+    \label{fig:dead_reckoning_example_2}
+\end{figure}
+In \autoref{fig:dead_reckoning_example_2} we receive a new update about the player's kinematic state. 
+This results in conflicting realities when compared to the dead reckoned position calculated from the previous example. 
+In the new update the player has turned right and a new dead reckoned position must be found, but this time it becomes a lot trickier since a believeable curve must be created from where we thought the player would be and where we estimate the player will be based on the new information. Representing the new position $P'_0$ immediately would result in the player warping across the screen which would not be ideal. Instead the player is represented at $ P_0 $ where we thought the player would be and the player should then move towards a new dead reckoned position, calculated with the information from the new update \autocite{DeadReckoning}.
+
+
+\subsection{Projective Velocity Blending}
+To calculate the curve that the player's movement needs to follow we need a good algorithm that is not too CPU intensive.
+The algorithm must work well for a segment of a curve that is passing through two points being the player's current position $P_0$ and the estimated future location $P'_1$.
+The recommened approach for this problem is projective velocity blending \autocite{DeadReckoning}.
+\\\\
+We create projections for the current kinematic state and the last known kinematic state and then these are blended together. 
+\begin{displaymath}
+    P_t = P_0 + V_0T_t + \frac{1}{2}A'_0T_t^2 \quad \rlap{\text{(Projecting from where player was)}}
+\end{displaymath}
+\begin{displaymath}
+    P'_t = P'_0 + V'_0T_t + \frac{1}{2}A'_0T_t^2 \quad \rlap{\text{(Projecting from last known)}}
+\end{displaymath}
+\begin{displaymath}
+    Q_t = P_t + (P'_t - P_t)\hat{T} \qquad \rlap{\text{(Combination of the two)}}
+\end{displaymath}
+\\\\
+Here, $T_t$ is how much time has elapsed since $T_0$ when is the time at the last known position, which would be the beginning of the curve. $T_1$ is the time when the player reaches the end of the curve. $\hat{T}$ is a normalized time value where $0.0\le\hat{T}\le1.0$ and should represent how far along the curve the player is.
```

Comment:
```suggestion
Here, $T_t$ is how much time has elapsed since $T_0$ which is the time at the last known position, which would be the beginning of the curve.
$T_1$ is the time when the player reaches the end of the curve.
$\hat{T}$ is a normalized time value where $0.0\le\hat{T}\le1.0$ and should represent how far along the curve the player is.
```